sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/OneCollectorChannelListener.java

```diff
@@ -118,7 +128,7 @@ public void onGroupRemoved(@NonNull String groupName) {
     public void onPreparedLog(@NonNull Log log, @NonNull String groupName) {
 
         /* Nothing to do on common schema log prepared. */
-        if (log instanceof CommonSchemaLog) {
+        if (!isOneCollectorCompatible(log)) {
```

Comment:
This decreases coverage, please update existing tests for that too.

Reply:
`OneCollectorChannelListener.java | 100% <100%> (ø)`